src/main/java/com/mycodefu/vsssfshss/server/MessageSender.java

```diff
@@ -5,4 +5,5 @@
 
 public interface MessageSender {
     void sendMessage(ChannelId channel, ByteBuf message);
+    void broadcast(ChannelId excludedChannelId, ByteBuf message);
```

Comment:
I find the restriction to one excluded channel to be a bit restrictive. Maybe we should use a vararg? Something like this?
```suggestion
    void broadcast(ByteBuf message, ChannelId... excludedChannels);
```
And we can use ChannelMatchers.compose in the NettyServer.